documents/dispute.md

```diff
@@ -0,0 +1,286 @@
+## Document
+
+### Fetch All Disputes
+
```

Comment:
Can you provide API docs in the PR description ?

Reply:
Pr description has been updated